.pre-commit-config.yaml

```diff
@@ -103,5 +93,15 @@ repos:
             metadata.kernelspec metadata.language_info.version
             cell.metadata.heading_collapsed metadata.name metadata.nbconvert_exporter
             metadata.version metadata.vscode
+  - repo: local
```

Comment:
is there a reason for this? is this commit noise?

Reply:
just moving pyright to run last to make it easier to surface pyright errors. Otherwise they get hidden by the output of the other precommits. See "along for the ride" section of the pr description